### PR TITLE
fedekunze/1648 gov params tab

### LINKED ISSUE #1648: Implement Governance Proposals Tab
Description:

Implement a governance Proposals tabs with the same design as the staking one

### PR DESCRIPTION
**Pull request by fedekunze**
Closes #1648

_Description:_

❤️ Thank you!

---

- [x] Added entries in `CHANGELOG.md` with issue # and GitHub username
- [x] Reviewed `Files changed` in the github PR explorer

### CODE REVIEW

File: app/src/renderer/components/governance/TabParameters.vue [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+  components: {
+    TmBtn,
+    TmListItem,
+    TmPage,
+    TmPart,
+    TmToolBar
+  },
+  data: () => ({
+    depositTooltips: {
+      description: `Parameters around proposals' deposits on governance`,
```
Comment (jbibla) [inline app/src/renderer/components/governance/TabParameters.vue]:
```suggestion
      description: `Governance parameters for the Cosmos Hub`,
```

File: app/src/renderer/components/governance/TabParameters.vue [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+      min_deposit: `Minimum deposit for a proposal to enter the voting period`,
```
Comment (jbibla) [inline app/src/renderer/components/governance/TabParameters.vue]:
```suggestion
      min_deposit: `Minimum deposit required before for a proposal can enter the voting period`,
```

File: app/src/renderer/components/governance/TabParameters.vue [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+      max_deposit_period: `Maximum period for token holders to deposit on a proposal`
```
Comment (jbibla) [inline app/src/renderer/components/governance/TabParameters.vue]:
```suggestion
      max_deposit_period: `Deposits must reach a "minimum deposit" in this period of time`
```

File: app/src/renderer/components/governance/TabParameters.vue [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+    },
+    tallyingTooltips: {
+      description: `Parameters around tallying votes in governance`,
+      threshold: `Percentage of Yes votes for proposal to pass`,
```
Comment (jbibla) [inline app/src/renderer/components/governance/TabParameters.vue]:
```suggestion
      threshold: `Percentage of "Yes" votes required for proposal to pass`,
```

File: app/src/renderer/components/governance/TabParameters.vue (hunk 1 above)
Comment (fedekunze) [reply app/src/renderer/components/governance/TabParameters.vue]: this is the description for the governance deposits params only

File: app/src/renderer/components/staking/TabParameters.vue [hunk 5]
```diff
@@ -14,7 +14,12 @@
           <div class="column">
             <dl class="info_dl">
               <dt>
-                Loose {{ parameters.parameters.bond_denom }}
+                Loose
+                {{
+                  stakingParameters.parameters.bond_denom
```
[the remark this answers is not in the document]
Comment (fedekunze) [reply app/src/renderer/components/staking/TabParameters.vue]: yeah if there's an error to display the params then the title won't be displayed completely

File: CHANGELOG.md [hunk 6]
```diff
@@ -39,6 +39,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - [\#1685](https://github.com/cosmos/voyager/pull/1685) added .idea to .gitignore @sabau
 - [\#1578](https://github.com/cosmos/voyager/issues/1578) added e2e tests for submit governance proposals @fedekunze
 - [\#1672](https://github.com/cosmos/voyager/issues/1672) added e2e tests for staking parameters tab @fedekunze
+- [\#1648](https://github.com/cosmos/voyager/issues/1648) added governance parameters tab @fedekunze
 - [\#1550](https://github.com/cosmos/voyager/issues/1578) added e2e tests for voting on governance proposals @fedekunze
```
[the remark this answers is not in the document]
Comment (fedekunze) [reply CHANGELOG.md]: thanks !

File: app/src/renderer/components/staking/TabParameters.vue (hunk 5 above)
[the remark this answers is not in the document]
Comment (jbibla) [reply app/src/renderer/components/staking/TabParameters.vue]: when would it be better to show `bondingDenom` over `stakingParameters.parameters.bond_denom`?

[the remark this answers is not in the document]
Comment (fedekunze) [reply app/src/renderer/components/staking/TabParameters.vue]: idk, is there a case on which the bondingDenom is not set? maybe we should check that and remove the hardcoded values from parameters everywhere

[the remark this answers is not in the document]
Comment (fedekunze) [reply app/src/renderer/components/staking/TabParameters.vue]: (on another PR)

[the remark this answers is not in the document]
Comment (faboweb) [reply app/src/renderer/components/staking/TabParameters.vue]: yes! can you create an issue?

**Pull request merged**